src/olympia/blocklist/cron.py

```diff
@@ -17,7 +17,8 @@ def upload_mlbf_to_kinto():
         log.info('Upload MLBF to kinto cron job disabled.')
         return
     log.info('Starting Upload MLBF to kinto cron job.')
-    server = KintoServer(KINTO_BUCKET, KINTO_COLLECTION_MLBF)
+    server = KintoServer(
+        KINTO_BUCKET, KINTO_COLLECTION_MLBF, kinto_sign_off_needed=False)
```

Comment:
nit: since the class is named `KintoServer`, does the argument name need the `kinto_` prefix?

Reply:
There's AMO sign off too so I felt it more explicit to name this arg `kinto_`